universal-application-tool-0.0.1/app/services/applicant/ApplicantServiceImpl.java

```diff
@@ -201,12 +202,20 @@ public ApplicantServiceImpl(
     return userRepository.programsForApplicant(applicantId);
   }
 
-  private void notifyProgramAdmins(long applicantId, long programId, String programTitle) {
-    String subject = String.format("New application submitted for %s", programTitle);
+  private void notifyProgramAdmins(long applicantId, long programId, String programName) {
+    String subject = String.format("New application submitted for %s", programName);
     String message =
         String.format(
             "Applicant %d submitted a new application to program %d", applicantId, programId);
-    amazonSESClient.send(PROGRAM_ADMIN_NOTIFICATION_MAILING_LIST, subject, message);
+    ImmutableList<Account> programAdmins = programRepository.getProgramAdministrators(programName);
+    programAdmins.forEach(
+        programAdmin -> amazonSESClient.send(programAdmin.getEmailAddress(), subject, message));
```

Comment:
You can reduce the number of calls by passing all the email addresses to the `send` call - that'll reduce the impact of incidental 500s, which happen sometimes.

We might also try catching errors here so that an applicant's submission works okay even if the emails don't send... or else maybe make the submission transactional and rollback if this fails?

Reply:
Will do, I am shamelessly copying your PR over.

Reply:
Done.